src/main/java/org/logstash/filters/NumericParserExecutor.java

```diff
@@ -0,0 +1,28 @@
+package org.logstash.filters;
+
+import org.joda.time.Instant;
+import org.logstash.Event;
+import org.logstash.filters.parser.TimestampParser;
+
+import java.io.IOException;
+
+class NumericParserExecutor implements ParserExecutor {
```

Comment:
The previous code in this PR used a bunch of if/else depending on an `instanceof` check. This separates the concern to individual classes instead. See https://github.com/logstash-plugins/logstash-filter-date/pull/74/commits/9ee4efd3e96045c03d6c4e67d6795990b280ed3e#diff-74eb22841ac6432cb296ec48b24b30efL94 for how it was done prior to NumericParserExecutor -- this was an improvement/refactoring by @guyboertje and discussed with me about it.